<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoystickStatusWindow</class>
 <widget class="QDialog" name="JoystickStatusWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Properties</string>
  </property>
  <property name="styleSheet">
   <string notr="true">JoyButtonStatusBox {
	border: 1px solid rgb(0, 0, 0);
}

JoyButtonStatusBox[isflashing=&quot;true&quot;] {
    background-color: rgb(0, 0, 255);
	color: rgb(205, 197, 191);
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Details</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>14</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="joystickNameLabel">
            <property name="text">
             <string>%1</string>
            </property>
            <property name="indent">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Number:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="joystickNumberLabel">
            <property name="text">
             <string>%1</string>
            </property>
            <property name="indent">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Axes:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="joystickAxesLabel">
            <property name="text">
             <string>%1</string>
            </property>
            <property name="indent">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Buttons:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="joystickButtonsLabel">
            <property name="text">
             <string>%1</string>
            </property>
            <property name="indent">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Hats:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="joystickHatsLabel">
            <property name="text">
             <string>%1</string>
            </property>
            <property name="indent">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10" stretch="2,1,1">
       <item>
        <widget class="QGroupBox" name="axesGroupBox">
         <property name="title">
          <string>Axes</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="bottomMargin">
           <number>14</number>
          </property>
          <item>
           <widget class="QScrollArea" name="axesScrollArea">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>350</width>
               <height>148</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="buttonsGroupBox">
         <property name="title">
          <string>Buttons</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="bottomMargin">
           <number>14</number>
          </property>
          <item>
           <widget class="QScrollArea" name="buttonsScrollArea">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>350</width>
               <height>72</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="hatsGroupBox">
         <property name="title">
          <string>Hats</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="bottomMargin">
           <number>14</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>JoystickStatusWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>JoystickStatusWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
